mixer/adapter/kubernetesenv/template/template.proto

```diff
@@ -72,6 +72,9 @@ message Template {
     // Destination pod's ip.
     istio.mixer.adapter.model.v1beta1.IPAddress destination_ip = 4;
 
+    // Destination container's port number.
+    int64 destination_port = 7;
```

Comment:
can you add a comment to the proto at the top to show last used field number / next available field number, as these are no longer in order?

Reply:
Done.